src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,161 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.domain.Dto.RankCountDto;
+import lotto.service.LottoService;
+import lotto.utils.SplitUtil;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoController {
+    private final LottoService lottoService;
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    public LottoController(InputView inputView, ResultView resultView, LottoService lottoService) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+        this.lottoService = lottoService;
+    }
+
+    public void run() {
+        PurchaseAmount purchaseAmount = createPurchaseAmount();
+        LottoTickets lottoTickets = createLottoTickets(purchaseAmount);
+        printLottoTickets(lottoTickets);
```

Comment:
```suggestion
        resultView.printLottoTickets(lottoTickets);
```
뷰에서 loop 는 뷰에서 해도 괜찮을것 같아요.

Reply:
해당 부분도 반영하여 수정하였습니다@